### PR TITLE
Add L1 to L2 upgrade decision tree

### PR DESCRIPTION
**Pull request by lvpeschke**
For anyone unsure about what they should be doing for the migration, a landing page to help understand next actions.

closes https://github.com/celo-org/celo-blockchain-planning/issues/572

### CODE REVIEW

**Comment by lvpeschke** [comment]
Closing in favour of https://github.com/celo-org/docs/pull/1466

File: docs/cel2/decision-tree.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  - [Infura](https://www.infura.io/networks/celo) (not yet verified)
+  - [DSRV](https://drpc.org/chainlist/celo) (not yet verified)
+
+
+  If none of the above work for you, you can use cLabs best effort hosted node: [Forno](https://docs.celo.org/network/node/forno).
+</details>
+<details>
+  <summary>I use ContractKit or Fee Abstraction (ERC-20 gas tokens)</summary>
+
+  Recommended versions for SDK and libraries:
```
Comment (GigaHierz) [inline docs/cel2/decision-tree.md:27]: If it's required, I would change it to "required"

File: docs/cel2/decision-tree.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  
+</details>
+
+<details>
+  <summary>I need help!</summary>
+
+  Reach out to us!
+
+  - Technical question? [celo-org discussions on GitHub](https://github.com/orgs/celo-org/discussions/categories/cel2)
+  - Any type of question? TODO Discord channel
```
Comment (GigaHierz) [inline docs/cel2/decision-tree.md:48]: please add. Please let me know if I should communicate that

Comment (GigaHierz) [reply docs/cel2/decision-tree.md:48]: also add link to Discord

**Review by jcortejoso — APPROVED** [review]

File: docs/cel2/decision-tree.md (hunk 1 above)
Comment (lvpeschke) [reply docs/cel2/decision-tree.md:27]: It's not, strictly speaking. The required versions are much lower, but @aaronmboyd made a good point in not encouraging people to use old versions at all and always point them to the latest.

File: docs/cel2/decision-tree.md (hunk 2 above)
Comment (lvpeschke) [reply docs/cel2/decision-tree.md:48]: The channel didn't exist at the time of writing 🙃 can add as soon as I can link to it

File: docs/cel2/decision-tree.md (hunk 1 above)
Comment (aaronmboyd) [reply docs/cel2/decision-tree.md:27]: That might have been @aaronmgdr hehe

Comment (lvpeschke) [reply docs/cel2/decision-tree.md:27]: True - Aarons everywhere 😆

**Pull request merged**